thoth/package_extract/image.py

```diff
@@ -546,6 +546,47 @@ def get_image_size(path: str) -> int:
     return total_size
 
 
+def _get_cuda_version(path: str) -> dict:
+    """Get the cuda version."""
+    res = dict()
+    # gathering version from version.txt file
+    version_path = path + "/usr/local/cuda/version.txt"
+    if os.path.isfile(version_path):
+        with open(version_path, "r") as f:
+            output = f.read()
+            lines = output.split()
+            if len(lines) != 3:
+                _LOGGER.info(
+                    "Unable to detect CUDA version from usr/local/cuda/version.txt output: %r",
+                    output,
+                )
+            else:
+                res["/usr/local/cuda/version.txt"] = lines[-1]
+
+    # gathering version from nvcc command
+    nvcc_path = glob.glob(path + '/usr/local/cuda-*/bin/nvcc')
+    if len(nvcc_path) > 0:
+        st = os.stat(nvcc_path[0])
+        os.chmod(nvcc_path[0], st.st_mode | stat.S_IEXEC)
+        result = run_command("{} --version".format(nvcc_path[0]), raise_on_error=False)
+        if result.return_code != 0:
+            _LOGGER.info(
+                "Unable to detect CUDA version - nvcc returned non-zero version: %s",
+                result.to_dict(),
+            )
+        else:
+            lines = result.stdout.splitlines()
+            version_info = lines[-1].split(",")
```

Comment:
So, we get the last line and split it by comma.

Reply:
Can we try to parse the line as above (expect the beginning of the line)? If NVIDIA changes the string in other releases, we might report wrong version with this assumption.